src/pages/[id]/index.js

```diff
@@ -50,11 +47,25 @@ const MemberProfile = ({ imageLink, user, contributions, errorMessage }) => {
     fillActiveTasksArray();
   }, [fillActiveTasksArray]);
 
+  if (errorMessage) {
+    return <NotFound errorMsg={errorMessage} />;
+  }
+
+  if (!isSuperUser) {
+    if (!apiCalledToVerifySuperUser) {
+      makeApiCallToVerifySuperUser();
+    }
+  }
+  if (!isSuperUser && !user.roles?.member) {
+    const errorMsg = `The Member Page for ${
+      id.charAt(0).toUpperCase() + id.slice(1)
```

Comment:
This seems a little out of the ordinary.
Why not use the first and last names here, and show userId as it is?

Reply:
Yeah, I have updated the code now to use the first name & last name.